helm/hpcc/values.yaml

```diff
@@ -38,6 +38,67 @@ global:
     moneyLocale: "en_US.UTF-8"
     # perCpu: 0.126
 
+##Minimal instance of Elastic stack will be deployed alongside HPCC components
+##Elastic stack is comprised of several components which follow below and can be customized
+##by overwriting/setting helm chart values here
+
+##The elasticsearch component can be customized by modifying helm chart values here.
+elasticsearch:
+  enabled: true
+  description: "HPCC Managed Elasticsearch"
+  ##See https://github.com/elastic/helm-charts/blob/master/elasticsearch/values.yaml for all available options
+  antiAffinity: "soft" #default is HARD, for minimal systems soft might be necessary
+  replicas: 2          #default is 3, for minimal systems 2 replicas might be adequate
```

Comment:
For minimal systems, 1 is adequate. Our default system does not provide redundancy for our own components, so providing it for elk seems anomolous

Reply:
@richardkchapman  Agreed, this issue kept me from bumping it down to 1: https://github.com/elastic/helm-charts/issues/783
I'm getting around it by removing pre-existing PVCs...